lib/beaker/logger.rb

```diff
@@ -89,6 +89,12 @@ def initialize(*args)
       dests.each {|dest| add_destination(dest)}
     end
 
+    # Setter for line_prefix that also calculates line_prefix_length
+    # @param [String] prefix The line prefix
+    def line_prefix=(prefix)
+      @line_prefix = prefix
+      @line_prefix_length = prefix.length
```

Comment:
@kevpl I was doing it for backwards compatibility bit it seems nothing is using it so i'll kill it